src/admin-views/order_modifiers/classic-fee-edit.php

```diff
@@ -26,36 +26,43 @@
 
 			<?php if ( ! empty( $automatic_fees ) ) : ?>
 				<div class="automatic-fees">
-					<strong><?php esc_html_e( 'The following fees will be automatically applied:', 'event-tickets' ); ?></strong>
-					<ul>
-						<?php foreach ( $automatic_fees as $automatic_fee ) : ?>
-							<li><?php echo esc_html( $automatic_fee->display_name ); ?></li>
-						<?php endforeach; ?>
-					</ul>
+					<strong><?php esc_html_e( 'The fees below automatically apply to all tickets.', 'event-tickets' ); ?></strong>
+					<?php foreach ( $automatic_fees as $fee ) : ?>
+						<div class="ticket_order_modifier_fees ticket_order_modifier_fees--single ticket_order_modifier_fees--automatic">
+							<input id="tec-ticket-fees-<?php echo esc_attr( $post_id ); ?>-<?php echo esc_attr( $fee->id ); ?>" value="<?php echo esc_attr( $fee->id ); ?>" type="checkbox" disabled="true" checked="true">
+							<label for="tec-ticket-fees-<?php echo esc_attr( $post_id ); ?>-<?php echo esc_attr( $fee->id ); ?>">
+								<?php echo esc_html( $fee->display_name . ' (' . $fee->raw_amount . ( $fee->sub_type === 'percent' ? '%' : '' ) . ')' ); ?>
+							</label>
+						</div>
+					<?php endforeach; ?>
 				</div>
 			<?php endif; ?>
 
 			<?php if ( ! empty( $selectable_fees ) ) : ?>
 				<div class="selectable-fees">
-					<select
-						class="tribe-dropdown"
-						name="ticket_order_modifier_fees[]"
-						id="ticket_order_modifier_fees"
-						multiple
-						data-placeholder="<?php esc_attr_e( 'Select fees...', 'event-tickets' ); ?>"
-						data-search-placeholder="<?php esc_attr_e( 'Search fees...', 'event-tickets' ); ?>"
-						data-allow-clear="true"
-						data-width="resolve"
-					>
-						<option value=""><?php esc_html_e( 'Select fees', 'event-tickets' ); ?></option>
+					<strong><?php esc_html_e( 'Below you can configure specific fees for this ticket.', 'event-tickets' ); ?></strong>
+					<div class="ticket_order_modifier_fees ticket_order_modifier_fees--selectable">
+						<select
+							class="tribe-dropdown"
+							name="ticket_order_modifier_fees[]"
+							id="ticket_order_modifier_fees"
+							multiple
+							data-placeholder="<?php esc_attr_e( 'Select fees...', 'event-tickets' ); ?>"
+							data-search-placeholder="<?php esc_attr_e( 'Search fees...', 'event-tickets' ); ?>"
+							data-allow-clear="true"
+							data-dropdown-css-width="false"
+							style="width: 100%;"
```

Comment:
We had to override this with an inline style?

Reply:
Unfortunately yes, thats how we do it in most of our codebase from what i found

Reply:
ugh, ok